dockerfiles/init/modules/openshift/files/scripts/che-openshift.yml

```diff
@@ -48,6 +48,10 @@ items:
       spec:
         containers:
         - env:
+          - name: OPENSHIFT_KUBE_PING_NAMESPACE
```

Comment:
Shouldn't be added the same changes to https://github.com/eclipse/che/blob/master/dockerfiles/init/modules/kubernetes/files/che-kubernetes.yaml? 
Also maybe kubernetes helm plugin should be updated accordingly.

@garagatyi @l0rd I assume it may affect minishift addon, should it be updated or the corresponding issue created?

Reply:
I can't find where this env var is used. @mshaposhnik @sleshchenko can you guys point me to it?

Reply:
Pls read https://github.com/jgroups-extras/jgroups-kubernetes/tree/0.9.3

Reply:
Thank you